blueprint/src/chapter/plancherel.tex

```diff
@@ -228,7 +228,7 @@ \section{\texorpdfstring{Plancherel's Theorem and the Fourier Transform on $L^2$
 such that $f_n\xrightarrow[L^2]{}f$. Such sequences exist:
 \begin{lemma}
   \label{lem:L12-dense}
-  \uses{MeasureTheory.Memℒp.exists_hasCompactSupport_snorm_sub_le}
+  %\uses{MeasureTheory.Memℒp.exists_hasCompactSupport_snorm_sub_le}
```

Comment:
What happened here?

Reply:
I've commented it out in order to avoid the `ERROR: Label '' could not be resolved` which is simply due to the fact that there isn't any previously defined label "MeasureTheory.Memℒp.exists_hasCompactSupport_snorm_sub_le" which can be used by that.

Reply:
Oh wait, this is a `uses` command, not a `lean` command. Makes sense.

Reply:
Yes, exactly.